pretix_eth/network/config.py

```diff
@@ -0,0 +1,14 @@
+"""
+Create a dictionary `networks_dict` with 
+key= network name (e.g. ZkSync, Loopring) and 
+value= object of the class implemented in networks.py
+"""
+import sys, importlib
+
+NETWORKS = ["L1", "Rinkeby", "ZkSync"]
+networks_dict = {}
+networks_module = importlib.import_module("pretix_eth.network.networks")
+
+for class_name in NETWORKS:
+    network_class_instance = getattr(networks_module, class_name)()
+    networks_dict[class_name] = network_class_instance
```

Comment:
I can totally see a use for `import_module` and `getattr` when we have some kind of pluggable system here and maybe a configuration in `settings` like:

```python
ENABLED_NETWORKS = {
    'rinkeby': 'path.to.rinkeby.module.AndClass',
    'zksync': 'another.path.to.module.ZkSyncClass',
}
```

But since we don't have anything like that just now - and who knows when/if we will, I'd vote to have pure imports here:

```python
from pretix_eth.network.networks import L1, Rinkeby, ZkSync

# if we need this configuration dict let's have it as well
networks = {
    'layer1': L1,
    'rinkeby': Rinkeby,
    'zksync': ZkSync,
}
```

I don't think there is anything wrong with your approach! I am just trying to increase the readability of the code.

Reply:
On the other hand I totally like the abstraction in the classes and I hope we will be able to keep that ;)

Reply:
Cool!
I was actually thinking to eventually use the `settings_from_fields()` function in `payments.py` - when creating the event, the admin would set which networks to add (along with ETH_RATE, DAI_RATE). Based on that, I was going to use `getattr` to instantiate the class.

But this hard-coding could work as well!